x-pack/plugins/security_solution/common/endpoint/schema/actions.ts

```diff
@@ -48,3 +48,19 @@ export const ActionDetailsRequestSchema = {
     action_id: schema.string(),
   }),
 };
+
+export const EndpointActionListRequestSchema = {
+  query: schema.object({
```

Comment:
shouldn't `query` also be `schema.maybe()`?  did you try to call the api with no params at all? just a `GET /api/endpoint/actions`?

Reply:
Yeah, it works without the query object. I've also added a test for this.